### PR TITLE
Cloud 8 Jenkins 

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: jenkins/ci.suse.de/cloud-mkcloud8.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          # - 'cloud-mkcloud{version}-job-btrfs-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar_register-{arch}'
+          #- 'cloud-mkcloud{version}-job-crowbar-devsetup-{arch}'
+          #- 'cloud-mkcloud{version}-job-dvr-{arch}'
+          #- 'cloud-mkcloud{version}-job-magnum-{arch}'
+          #- 'cloud-mkcloud{version}-job-monasca-{arch}'
+          #- 'cloud-mkcloud{version}-job-raid-{arch}'
+          #- 'cloud-mkcloud{version}-job-ssl-{arch}'
+          #- 'cloud-mkcloud{version}-job-upgrade-disruptive-{arch}'
+          #- 'cloud-mkcloud{version}-job-xen-{arch}'
```
Comment (jsuchome) [inline jenkins/ci.suse.de/cloud-mkcloud8.yaml]: Do we need them commented? It's saved elsewhere, so let's rather keep this file clean...

File: jenkins/ci.suse.de/cloud-mkcloud8.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        - 'cloud-mkcloud{version}-job-ha-compute-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-linuxbridge-{arch}'
+        - 'cloud-mkcloud{version}-job-ha-ironic-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-disruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-ceph-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-ceph-compute_ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-dvr-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-lbaasv2-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-without-nodes-upgrade-{arch}'
```
Comment (jsuchome) [inline jenkins/ci.suse.de/cloud-mkcloud8.yaml]: I do not think there's much value in keeping these upgrade jobs here (even with `disabled: true`). What about just enabling the project with one simple HA job?

**Comment by jsuchome** [comment]
Don't we need to copmy templates as well?
Also, you're keeping ironic scenario, but ironic job is disabled...

**Comment by dirkmueller** [comment]
@jsuchome cleaned up, added ironic job

**Pull request merged**